examples/fixtures/unittests/pyproject-unknown.toml

```diff
@@ -0,0 +1,2 @@
+[build-system]
```

Comment:
Do we need this file, too?

Reply:
Yes, it is needed by python unit tests. Moved it too.